cd/python/pypi/pypi_package.sh

```diff
@@ -22,11 +22,10 @@ set -ex
 export mxnet_variant=${1:?"Please specify the mxnet variant"}
 
 # Due to this PR: https://github.com/apache/incubator-mxnet/pull/14899
-# The setup.py expects that mkldnn_version.h be present in
+# The setup.py expects that dnnl_version.h be present in
 # mxnet-build/3rdparty/onednn/build/install/include
 # The artifact repository stores this file in the dependencies
 # and CD unpacks it to a directory called cd_misc
-# Nov. 2019 Update: With v1.1, MKL-DNN is renaming to DNNL. Hence changing the prefix of file name.
 if [ -f "cd_misc/dnnl_version.h" ]; then
   mkdir -p 3rdparty/onednn/include/oneapi/dnnl
   cp cd_misc/dnnl_version.h 3rdparty/onednn/include/oneapi/dnnl/.
```

Comment:
check directory here onednn dnnl ?

Reply:
I do not understand.

Reply:
sorry, my fault, please ignore this comment